tests/web/rpc/v1/routing_strategies/test_outcomes_based.py

```diff
@@ -47,6 +47,25 @@ def _get_request_meta(
     )
 
 
+def _get_highest_accuracy_request_meta(
```

Comment:
nit: you could just add an argument to `_get_request_meta` with the default being NORMAL and save 10 lines of code